packages/as-proba/assembly/probability/sampler.ts

```diff
@@ -101,7 +119,10 @@ export class Sampler {
   }
 
   /**
-   * Populate observation zone bounderies.
+   * This method populates observation zone bounderies.
```

Comment:
```suggestion
   * This method populates observation zone boundaries.
```

Reply:
There are a few more instances of "bounderies" to be replaced by "boundaries at the end of the file.